test/controllers/MovementsControllerSpec.scala

```diff
@@ -196,6 +242,47 @@ class MovementsControllerSpec extends SpecBase with ScalaCheckPropertyChecks wit
           verify(mockArrivalMovementRepository, times(0)).insert(any())
         }
       }
+
+      "must return InternalServerError if sending the message upstream fails" in {
```

Comment:
It would be good to know if the error is within our service or EIS (i know you'd mentioned something about adding specific logging for this). Just something to keep in mind